kong/client_test.go

```diff
@@ -199,3 +199,23 @@ func TestTestWorkspace(T *testing.T) {
 	assert.NoError(err)
 	assert.Equal(currWorkspace.Config, origWorkspace.Config)
 }
+
+func TestBaseRootURL(t *testing.T) {
+	t.Run("default", func(t *testing.T) {
+		client, err := NewClient(nil, nil)
+		require.NoError(t, err)
+		require.NotNil(t, client)
+
+		require.Equal(t, client.BaseRootURL(), "http://localhost:8001")
+	})
+
+	// Don't test setting the URL via env because that can mess up other tests.
```

Comment:
We can consider using `t.SetEnv` (it ensures that the test using it is not parallel so should be safe to use): 
https://cs.opensource.google/go/go/+/refs/tags/go1.19.4:src/testing/testing.go;l=1287

Reply:
Added. Thanks, one learns something every day!